pages/linux/pw-cli.md

```diff
@@ -0,0 +1,16 @@
+# pw-cli
+
+> The PipeWire Command Line Interface.
+> More information: <https://docs.pipewire.org/page_man_pw_cli_1.html>.
```

Comment:
@navarroaxel, @CleanMachine1, what about using [manned.org][doc]? The provided link referrers to a page with not user-friendly formatted page

[doc]: https://manned.org/pw-cli.1#head6